output/json/json.go

```diff
@@ -32,7 +32,6 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-// TODO: add option for emitting proper JSON files (https://github.com/loadimpact/k6/issues/1052)
```

Comment:
Ah, this was a wrong copy-paste on my part, the actual issue in the brackets here should have been https://github.com/loadimpact/k6/issues/737, which is still not fixed. Can you please add the `TODO` back, but with the correct link this time? :sweat_smile:

Reply:
@na-- Done